src/index.ts

```diff
@@ -1,10 +1,6 @@
-import { startPublishingTestPings } from './server/graphql/graphql'
 import { startChessServer } from './server/server'
```

Comment:
I like the idea, that the main file has the same name as the package, because of the problem with many files having the same name, but I think the imports look like they're importing some internal module they shouldn't. 

In Python you're supposed to keep the __init__.py files lightweight and mostly just re-export stuff from other files (something I just learned during last weeks exercises, I used to lot of code there too). I think the same could apply here.